apps/web-staking/.gitignore

```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 .yarn/install-state.gz
+/.vscode
```

Comment:
Why is this added here?

Reply:
Needed to be added because it was not in the web-staking until now and we don't want to commit the .vscode yet